python/lsst/verify/spec/base.py

```diff
@@ -139,19 +139,36 @@ def check(self, measurement):
         """
         pass
 
-    def query_metadata(self, metadata):
+    def query_metadata(self, metadata, arg_driven=False):
         """Query a Job's metadata to determine if this specification applies.
 
         Parameters
         ----------
         metadata : `lsst.verify.Metadata` or `dict`-type
             Metadata mapping. Typically this is the `lsst.verify.Job.meta`
             attribute.
+        arg_driven : `bool`, optional
```

Comment:
Can you use `functools.wraps` to inherit the docstrings.  I'm always nervous when I see this much text duplicated in two places.